scripts/version_mapper.py

```diff
@@ -130,7 +130,7 @@ def update_switch_version_file(
                     print(f"Writing content to {announcement_file}.")
                     file.write(announcement_content)
 
-    # Make the redirec page to point to the latest stable
+    # Make the redirect page to point to the latest stable
     with open("index.html", "r") as redirection_file:
         content = redirection_file.read()
     new_content = content.replace("var-url", f"{cname}/release/{latest_stable_version}")
```

Comment:
Let us also add it in here, just for making everything uniform:

```suggestion
    new_content = content.replace("var-url", f"{cname}/release/{latest_stable_version}/")
```